daml-lf/transaction/src/main/protobuf/com/daml/lf/transaction.proto

```diff
@@ -126,4 +126,17 @@ message NodeRollback { // *since version 14*
     repeated string children = 1; // node ids
 }
 
+message FatContractInstance {
+  bytes contract_id = 1;
+  com.daml.lf.value.Identifier template_id = 2;
+  bytes arg = 3;
+  bytes key = 4;
```

Comment:
It was designed as an optimization to avoid decoding and validations of the of the disclosed contracts in case they  are not use. 
For now we do not use it at all, but we just verified it is correct. 
Taking into account we can recompute it (without the package) and we can (relatively) easily extend the format of the blob, I suggest we disregard the hash.